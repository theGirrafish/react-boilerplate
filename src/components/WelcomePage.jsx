import React, {PureComponent} from 'react';
import Logo from '../assets/images/react.png';

class WelcomePage extends PureComponent {
  constructor(props) {
    super(props);
    this.state = {
    };
  }

  render() {
    return (
      <div className='background'>
        <img className='react_logo' src={Logo}/>
        <div className='text'>
          <span className='title'>The React Boilerplate</span>
          <span className='subtitle'>Congratulations!</span>
          <span className='subtitle'>Looks like everything is up and running.</span>
        </div>
      </div>
    );
  }
}

export default WelcomePage;
