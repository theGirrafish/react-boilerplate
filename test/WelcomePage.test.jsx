import React from 'react';
import renderer from 'react-test-renderer';
import WelcomePage from '../src/components/WelcomePage';

describe('WelcomePage', () => {
  test('snapshot test', () => {
    const snapshot = renderer.create(<WelcomePage/>).toJSON();
    expect(snapshot).toMatchSnapshot();
  });
});
