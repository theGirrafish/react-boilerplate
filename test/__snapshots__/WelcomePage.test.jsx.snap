// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage snapshot test 1`] = `
<div
  className="background"
>
  <img
    className="react_logo"
    src={[Function]}
  />
  <div
    className="text"
  >
    <span
      className="title"
    >
      The React Boilerplate
    </span>
    <span
      className="subtitle"
    >
      Congratulations!
    </span>
    <span
      className="subtitle"
    >
      Looks like everything is up and running.
    </span>
  </div>
</div>
`;
